### PR TITLE
Ruby 2.5.3 upgrade

### PR DESCRIPTION
**Pull request by ztmeyer**

### CODE REVIEW

File: .ruby-version [hunk 1]
```diff
@@ -0,0 +1 @@
+2.5.3
```
Comment (wpliao1989) [inline .ruby-version]: Super minor. Missing new line.

File: .travis.yml [hunk 2]
```diff
@@ -1,4 +1,4 @@
 language: ruby
 rvm:
-  - 2.2.4
-before_install: gem install bundler -v 1.11.2
+  - 2.5.3
```
Comment (wpliao1989) [inline .travis.yml]: Can we delete this file and get rid of TravisCI? (This is a question, I don't know how travis is set up for this)

File: .circleci/config.yml [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+      - run: bundle install
+      - run: bundle exec appraisal install
+      - run: bundle exec appraisal rake test
+
+jobs:
+  test-ruby-253:
+    steps:
+    - bundle_install_and_test
+    docker:
+    - image: circleci/ruby:2.5.3-stretch
```
Comment (wpliao1989) [inline .circleci/config.yml]: What does `-stretch` image do? Normally just `circleci/ruby:2.5.3` should work.

**Review by wpliao1989 — APPROVED** [review]
🚀

File: .travis.yml (hunk 2 above)
Comment (ztmeyer) [reply .travis.yml]: TravisCI says this is "not an active repository" and there are no branches/pulls/builds. I'll remove this.

**Pull request merged**